aws_doc_sdk_examples_tools/config/services.yaml

```diff
@@ -2424,6 +2424,22 @@ memorydb:
   tags:
     product_categories: {}
   version: memorydb-2021-01-01
+neptune:
+  long: '&neptunelong;'
+  short: '&neptune;'
+  sort: Forecast
+  sdk_id: forecast
+  expanded:
+    long: Amazon Neptune Service
+    short: Neptune
+  blurb: is a fully managed graph database service by AWS, designed specifically for handling complex relationships and connected datasets at scale.
```

Comment:
This should come from the landing page.

Reply:
I updated the blurb with information from the landing page.